README.md

```diff
@@ -29,17 +29,20 @@ If you are using Maven to manage your project's dependencies, you can run `mvn i
 If you aren't using Maven, you can add the `jar` in the normal way.  This `jar` can be found in
 the `target` directory created when you ran `mvn package`.
 
-Note that this will *not* run the integration tests, since these require access to a Conjur instance.  To run the
-integration tests, you will need to define the following environment variables for the `mvn package` command 
-(and remove the `skipTests` property):
+Note that this *did not* run the integration tests, since these require access to a Conjur instance. You can run the 
```

Comment:
What does the word "this" in the "Note that this" sentence refer to?

Reply:
Good call, it made sense once but i changed the text.

I'll change the sentence to: Note that we ran `mvn package` without running the integration tests, ...